modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -47,6 +47,28 @@ In the following example, the app will perform an `ADD_TASK` mutation which will
 
 For more information, see link:https://www.apollographql.com/docs/react/essentials/mutations.html#props[Apollo's document about mutations].
 
+== Offline Workflow
+
+When offline `client.mutate` function will return immediately after is called.
```

Comment:
Proposal: "When the {sync-client} changes to an offline state, the `client.mutate` function is immediately triggered and  results in an error message being captured. Developers can use this message to determine what the issue is, and write code to handle this situation on the server."

Reply:
I have tried to apply this but there are some technical problems in the sentence. How about:

`When the mutation is executed while the application is in the ofline state, the client.mutate function **will return** immediately and *return* Promise with an error. Developers can use this *error* object to determine if the error is related to offline state and monitor when a change will be synced back to server`

Reply:
This is better, but would like to rewrite (this is important for offline,right?):

If a mutation occurs while the device is offline, the client.mutate function:
* returns immediately
* returns a promise with an error
You can use check error objects to isolate errors relating to offline state and sync the change back to the server, when the device is online again.

Kinda begs for an example, right? maybe a link to another section or showcase app

Reply:
@finp WOW. Really really like it! Doing chores now.

Reply:
Forgot to mention. An example is in code straight below these sentences.